api/uploader/upload.go

```diff
@@ -40,6 +44,7 @@ func New() (*API, error) {
 	return &API{
 		Config: *c,
 		client: http.Client{},
+		Logger: logger.New(),
```

Comment:
Upload constructor is used only in tests so we can't pass Logger from Cloudinary here.